public/data/ShlokMohanty.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub",
+  "avatar": "https://github.com/ShlokMohanty.png",
+  "links": [
+    {
+      "name": "Follow me on github",
+      "url": "https://github.com/ShlokMohanty",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mohanty_shlok",
+      "icon": "twitter"
+    },
+   {
+    "name": "Follow me on Facebook",
+    "url": "https://www.facebook.com/shlok.mohanty.35",
+    "icon": "facebook"
```

Comment:
Add 2  white space before these dict variables

Reply:
Line number 17, 18, and 19

Reply:
@FayasNoushad made the changes can you see it in the pr generated ?